electron/preload.ts

```diff
@@ -30,16 +30,166 @@ const api = {
   Close: () => {
     ipcRenderer.send('close');
   },
+  removeLoading: () => {
+    // TO-DO
```

Comment:
If not needed , than rmeove this

Reply:
removeLoading function was implemented below, I changed the order of the declaration to remove the TO-DO comment